backend/src/Equinor.ProjectExecutionPortal.WebApi/Controllers/ProfileController.cs

```diff
@@ -1,32 +0,0 @@
-﻿using Equinor.ProjectExecutionPortal.WebApi.Authorization.Extensions;
```

Comment:
Kan vi ta denne deleten in en egen pr? ettersom en da ikke knekker admin appen?

Reply:
Enig. Har lagt ProfileController inn igjen.